packages/network-score/scoring.go

```diff
@@ -0,0 +1,215 @@
+package network_score
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/kartikaysaxena/cord.go/packages/did"
+	identifier "github.com/kartikaysaxena/cord.go/packages/identifier/src"
```

Comment:
Why are these being imported under `karthikaysaxena`.? 
If you require some changes to be made to the upstream and then be used. 
Please ask, I would fork upstream under `dhiway`, where you would make changes required.

Reply:
Actually I made the sdk and tried to publish it and import it remotely to check if it working, in Go, the library names are based on the Github URIs that we publish the code into, changing this to `dhiway/cord.go` would be simple, would do that.